Tools/buildstream/elements/sdk/icecc.bst

```diff
@@ -9,6 +9,7 @@ depends:
 - freedesktop-sdk.bst:components/libcap-ng.bst
 - freedesktop-sdk.bst:components/libarchive.bst
 - freedesktop-sdk.bst:components/lzo.bst
+- freedesktop-sdk.bst:components/openssl.bst
```

Comment:
Adding a dependency to the `icecc` build recipe seems unrelated to the rest of the patch. Let's remove this change before landing, then add it in a separate patch.

Reply:
It is related to the junction bump... upstream SDK updated their libarchive version and now without this icecc configure fails to find libssl and libcrypto.

Reply:
Got it, thanks for commenting on this.